extension/js/common/assert.ts

```diff
@@ -37,7 +37,7 @@ export class Assert {
     if (acctEmail) {
       const [primaryKi] = await KeyStore.get(acctEmail, ['primary']);
       const { setup_done } = await AcctStore.get(acctEmail, ['setup_done']);
-      if (setup_done && primaryKi && !(await KeyUtil.parse(primaryKi.private)).fullyEncrypted) {
+      if (setup_done && primaryKi && !(await KeyUtil.parse(primaryKi.private))?.fullyEncrypted) {
```

Comment:
This looks suspect - if there is primaryKi, then KeyUtil parse should always return it. I'll have to look why is it Optional, and either bang! it (I'd rather it fails) or address it differently.

Reply:
You can remove it and check yourself - it throws errors when a user closes google auth modal. I'm not familiar with the project so close to fix it properly, that's why I did it like this. If you have other ideas, please let me know.